upgrade/UPGRADE-v8.1.2.md

```diff
@@ -0,0 +1,40 @@
+# [Upgrade from v8.1.1 to v8.1.2](https://github.com/shopsys/shopsys/compare/v8.1.1...v8.1.2)
+
+This guide contains instructions to upgrade from version v8.1.1 to v8.1.2.
+
+**Before you start, don't forget to take a look at [general instructions](/UPGRADE.md) about upgrading.**
+There you can find links to upgrade notes for other versions too.
+
+## [shopsys/framework]
+
+[shopsys/framework]: https://github.com/shopsys/framework
+
+### Application
+
+- update selling to for product data fixture in order to fix failing tests (#1677)
+  ```diff
+      # ProductDataFixture.php
+      - $this->setSellingFrom($productData, '11.2.2020');
+      + $this->setSellingFrom($productData, '11.2.2320');
+      ```
+  see project-base diff to update your project
+
+- pin version of `fp/jsformvalidator-bundle` to `~1.5.1` as next minor is not meant to be used with Symfony 3 ([#1790](https://github.com/shopsys/shopsys/pull/1790))
```

Comment:
I am just guessing, but should this line mention Symfony 4, instead of Symfony 3?